### ISSUE TITLE
Support SVG in image()

### ISSUE THREAD

**Issue by Lorp** [typemytype/drawbot#240]
Any chance that SVG images can be supported in the image() function?

**Comment by justvanrossum** [comment]
The problem was always that SVG is not directly supported at the Quartz level. Not sure whether that situation has improved with 10.13 or 10.14.

**Comment by gferreira** [comment]
I’ve managed to use [Simple SVG Parser](https://github.com/evanw/simple_svg_parser) some time ago to create a custom pen to convert flat SVGs into glyphs. the same could work for DrawBot if we add support for colors…

**Comment by typemytype** [comment]
in cocoa a WebKit htmlView can render a svg, convert that to a pdf and use that data in an image. This feels like a big detour ;)

**Comment by gferreira** [comment]
ok, goed to know. but in the meantime, the detour works!

![screen shot 2018-11-13 at 13 56 42](https://user-images.githubusercontent.com/62717/48425497-17f0fc00-e74c-11e8-8f4b-ef6e399bb435.png)

**Comment by typemytype** [comment]
oh cool could you post this on the drawbot forum?

thanks

**Comment by gferreira** [comment]
I’ve started to convert my quick & dirty example into a more useful [conversion tool](http://github.com/gferreira/svg2drawbot/), separating parsing from drawing. (still a work in progress)

I’ve tested it with different SVGs, and not all of them work: `simple_svg_parser` supports only `px` as units, but I’ve found lot of SVGs defined in `%` or `cm`.

there also seems to be a problem with shapes which have a stroke but no fill (it needs to be set to `None` explicitly).  for example, this is what I get when trying to draw the [GhostScript Tiger](https://commons.wikimedia.org/wiki/File:Ghostscript_Tiger.svg):

![screen shot 2018-11-15 at 09 09 07](https://user-images.githubusercontent.com/62717/48549367-bef1a700-e8b6-11e8-9d14-f37aeee4c191.png)

btw thanks @typemytype for feedback on the current code – I will be making some changes soon.

**Comment by gferreira** [comment]
@justvanrossum I agree. this is meant only as a helper to get things done today. a bird in the hand…

**Comment by Lorp** [comment]
Good discussion, thanks for the explanations and workaround.
